mars/core.py

```diff
@@ -667,6 +667,16 @@ class TileableOperandMixin(object):
     def check_inputs(self, inputs):
         pass
 
+    def _check_if_gpu(self, inputs):
```

Comment:
May change to ``_fill_if_gpu`` as the function changes the attributes of entities.

Reply:
I keep the function name, and return value in it.